examples/evaluate.py

```diff
@@ -185,23 +191,32 @@ def evaluate_replicate(
     """
     # Dataset will only be downloaded if it does not exist
     subset: WILDSSubset = dataset.get_subset(split)
-    true_labels: torch.Tensor = subset.y_array
     metadata: torch.Tensor = subset.metadata_array
-    # predicted_labels.resize_(true_labels.shape)
+    true_labels = subset.y_array
     if predicted_labels.shape != true_labels.shape:
         predicted_labels.unsqueeze_(-1)
     return dataset.eval(predicted_labels, true_labels, metadata)[0]
 
 
-def get_predictions(path: str) -> List[Any]:
+def evaluate_replicate_for_globalwheat(
+    dataset: WILDSDataset, split: str, path_to_predictions: str
+) -> Dict[str, float]:
+    predicted_labels = torch.load(path_to_predictions)
+    subset: WILDSSubset = dataset.get_subset(split)
+    metadata: torch.Tensor = subset.metadata_array
+    true_labels = list(itemgetter(*subset.indices)(subset.dataset.y_array))
```

Comment:
Hmm do we need something so complicated? I think the standard way would be to do a list comprehension?

Reply:
Do you mean something like this: `subset.dataset.y_array[subset.indices]`? This give me an error: *** TypeError: only integer scalar arrays can be converted to a scalar index, so I used `itemgetter` as a workaround.